src/assets/scripts/install-wsl-helpers

```diff
@@ -12,3 +12,6 @@ set -o errexit -o nounset
 mkdir -p "/mnt/wsl/rancher-desktop/bin/"
 cp "${1}" "/mnt/wsl/rancher-desktop/bin/nerdctl"
 chmod u+s "/mnt/wsl/rancher-desktop/bin/nerdctl"
+if [ ! -f "/mnt/wsl/rancher-desktop/bin/docker" ] ; then
+  ln -s "/mnt/wsl/rancher-desktop/bin/nerdctl" "/mnt/wsl/rancher-desktop/bin/docker"
```

Comment:
Well, I used an absolute symlink to be consistent with `.../resources/darwin/bin/docker`, which is created as a symlink to the absolute path. If it's a relative path, then `dirname $(readlink ${BASH_SOURCE[0]})` resolves to `.`, and then looking for `limactl` fails as it resolves to `../lima/bin/limactl` instead of `RDHOME/resources/darwin/lima/bin/limactl`.

This was with a dev build -- I didn't bother trying this with a prod build.